src/controller/schema_controller.ts

```diff
@@ -22,18 +24,20 @@ export async function createSchema(
       await addDelegateAsRegistryDelegate();
     }
 
-    const data = req.body.schema;
-
-    if (!data || !data.properties) {
-      return res.status(400).json({
-        error:
-          "'schema' is a required field in the form of key-value pair, with title and description",
-      });
-    }
+    let data = req.body.schema?.schema || req.body.schema || null;
 
+    
+    const validationError = validateSchema(data);
+    if (validationError) {
+      return res.status(400).json({ error: validationError });
+        }
+    
+    data = extractSchemaFields(data)
```

Comment:
do you want to take it as a separate variable? 

It may help in debug and other reasons later.

Reply:
If I treat it as a separate variable, I will encounter a type error when passing it to the buildFromProperties function, indicating that the following properties are missing from the type 'ISchema': $id, $schema